test/parallel/test-crypto-sign-verify.js

```diff
@@ -70,3 +70,10 @@ const keyPem = fs.readFileSync(common.fixturesDir + '/test_key.pem', 'ascii');
   verified = verStream.verify(certPem, s3);
   assert.strictEqual(verified, true, 'sign and verify (stream)');
 }
+
+// Test throws exception when key options is null
+{
+  assert.throws(() => {
+    crypto.createSign('RSA-SHA1').update('Test123').sign(null, 'base64');
+  }, /No key provided to sign/);
```

Comment:
Can you change the regular expression to match the full error message? `/^Error: No key provided to sign$/`